lib/api/1.1/southbound/templates.js

```diff
@@ -70,7 +74,7 @@ function templatesRouterFactory (
             presenter(req, res)
                 .renderTemplate(
                     req.params.identifier,
-                    properties.options,
+                    _.merge(properties.options, {identifier: nodeId}),
```

Comment:
How about setting the key as `nodeId` rather than `identifier`, as the "identifier" is a so general word.